### ISSUE TITLE
Ore Stone Variants Makes it so Aether Ores Don't Spawn at all

### ISSUE THREAD

**Issue by EStuffs** [PersonTheCat/ore_stone_variants#82]
![image](https://user-images.githubusercontent.com/38076525/67623891-341c6a80-f7f0-11e9-9d04-99729a4646cc.png)
When Ore Stone Variants and Aether are both installed, the aether ores just don't spawn at all, including ice stone and moss stone
i've tried using ore stone and aether alone as well, and have confirmed it is this mod compatibility, I have no idea what's going on with this, and we have been having this issue for multiple weeks, the mod pack we are using is Era of Magic 2 (Community Edition)

**Comment by PersonTheCat** [comment]
Hey, there. Thanks for reporting this! This issue has been around since an earlier version of the mod from last year. I'll try to look into it soon, now that the 1.14 beta is out.

**Comment by EStuffs** [comment]
> 
> 
> Hey, there. Thanks for reporting this! This issue has been around since an earlier version of the mod from last year. I'll try to look into it soon, now that the 1.14 beta is out.

this is 1.12.2 btw

**Comment by PersonTheCat** [comment]
Any way you could try this new build for me? I just updated to 1.12.2 so I could test this again directly from my IDE and can no longer reproduce it. Just change the extension from `.zip` to `.jar`.

[ore_stone_variants-4.6.zip](https://github.com/PersonTheCat/ore_stone_variants/files/3800712/ore_stone_variants-4.6.zip)

**Comment by EStuffs** [comment]
I'll make sure to check it out later today




Sent via the Samsung Galaxy S9, an AT&T 5G Evolution capable smartphone


Get Outlook for Android







On Sat, Nov 2, 2019 at 11:40 AM -0500, "PersonTheCat" <notifications@github.com> wrote:












Any way you could try this new build for me? I just updated to 1.12.2 so I could test this again directly from my IDE and can no longer reproduce it. Just change the extension from .zip to .jar.


ore_stone_variants-4.6.zip

**Comment by EStuffs** [comment]
> 
> 
> Any way you could try this new build for me? I just updated to 1.12.2 so I could test this again directly from my IDE and can no longer reproduce it. Just change the extension from `.zip` to `.jar`.
> 
> [ore_stone_variants-4.6.zip](https://github.com/PersonTheCat/ore_stone_variants/files/3800712/ore_stone_variants-4.6.zip)

it didn't work, still no aether ores

**Comment by Metadorius** [comment]
@PersonTheCat the issue is still there as of latest A:L 1.4.4 and Ore Stone Variants 5.1. This is a dealbreaker for me because in my private modpack the Aether's ores are a part of progression. Thanks.

**Comment by Handeril** [comment]
Aether: Legacy's Github was recently archived by it's owner, so it might be safe to say now that Aether: Legacy will not be receiving another update in the near future. If this bug is patched in the current 1.12.2 version, please let me know.

I would like to look into using this mod in my upcoming modpack, mainly for adding additional Basalt/Limestone/Marble/etc variants of ores.

**Comment by PersonTheCat** [comment]
Yeah, it's a pretty complicated fix in 1.12. Forge didn't use to let you discriminate which _kind_ of custom ores you're disabling. There's not much I can actually do to fix this without adding a dependency on Mixin. It would be a pretty significant change for 1.12, so I just won't have time for that anytime soon. :(

**Comment by Handeril** [comment]
I look forward to if anyone wants to take up the challenge, or when you might have the time yourself to work on this.

The modpack I'm working on isn't a "one and done" sort of deal, I hope to get it published soon, but I will still be working on it afterwards.
